packages/content-management/table_list_view_table/src/components/table.tsx

```diff
@@ -298,7 +301,14 @@ export function Table<T extends UserContentCommonSchema>({
         selection={selection}
         search={search}
         executeQueryOptions={{ enabled: false }}
-        sorting={tableSort ? { sort: tableSort as PropertySort } : undefined}
+        sorting={
+          tableSort
```

Comment:
Some things catch my attention here:
1. If the typings are correct, then `tableSort` will never be undefined or null, but it is treated as though it could be.
2. Casting `tableSort as PropertySort` doesn't seem to be necessary

I think pulling the value out into a variable, and simplifying per the above, will help:
```
  const sorting =
    tableSort.field === 'accessedAt' // "accessedAt" is a special case with a custom sorting
      ? true // by passing "true" we disable the EuiInMemoryTable sorting and handle it ourselves
      : { sort: tableSort };
```